### ISSUE TITLE
Help localization of NVDA in new languages by adding the missing translators comments

### ISSUE THREAD

**Issue by JulienCochuyt** [nvaccess/nvda#11104]
NVDA is localized (translated in 70 languages) using GNU gettext.

From [NVDA wiki Contributing / Translatable Strings](https://github.com/nvaccess/nvda/wiki/Contributing#translatable-strings):

> All translatable strings should have a preceding translators comment describing the purpose of the string for translators. For example:
> ```
> # Translators: The name of a category of NVDA commands.
> SCRCAT_TEXTREVIEW = _("Text review")
> ```

For new contributions, this obligation is ensured by the `checkPot` test.
Still, there are dozens of translatable strings that lack such a comment in code contributed before the creation of this test.

A complete list of these can be found near the top of `tests/checkPot.py`.

All these strings are already translated in the current 70 languages in which NVDA is available, but the lack for translators comment makes it uselessly harder to add a new language to the list.

As stated by @feerrenrut in https://github.com/nvaccess/nvda/pull/11093#discussion_r419287528, It would be greatly appreciated if some good willing volunteer took the charge to add all or part of these missing translators comment.

**Comment by josephsl** [comment]
Hi, note that some of them will involve creating multiple translator comments for the same string (I expect pgettext might mitigate it somewhat as it can provide contextual comments). For past precedence, see #1410 (nvdakor is me). Thanks.

**Comment by Adriani90** [comment]
Is this still relevant for translators?
cc: @michaelDCurran, @CyrilleB79

**Comment by seanbudd** [comment]
Yes there are still many missing translator comments

**Comment by CyrilleB79** [comment]
Yes, translators still use these comments.

**Comment by rrajput22** [comment]
Hi! I’d like to work on this issue. I will add translator comments for several of the missing strings listed in tests/checkPot.py. I’ll submit a PR with a small batch first so maintainers can confirm the format.

**Comment by seanbudd** [comment]
Thank you @rrajput22
